### PR TITLE
Refs [39250] Tạo trang Blog

### PR DESCRIPTION
**Pull request by NDPNguyen-FE**
https://edu-redmine.sun-asterisk.vn/issues/39250

### CODE REVIEW

**Comment by NDPNguyen-FE** [comment]
ready

File: src/pug/component/cardBlog.pug [hunk 1]
```diff
@@ -1,5 +1,5 @@
 mixin cardBlog(src, alt, title, sub, desc)
-    .cardBlog.text-left.col-md-6.col-12
+    .cardBlog.text-left
         .cardBlog__image
```
Comment (vietnq-1097) [inline src/pug/component/cardBlog.pug]:
```suggestion
    .card-blog.text-left
        .card-blog__image 
```
để snake thay vì camelCase

**Comment by NDPNguyen-FE** [comment]
ready

**Comment by NDPNguyen-FE** [comment]
ready

**Comment by NDPNguyen-FE** [comment]
ready

**Pull request merged**